packages/vx-shape/src/types/accessor.ts

```diff
@@ -0,0 +1,9 @@
+export type Accessor<Datum, Output> = (d: Datum) => Output;
```

Comment:
💬  All accessor types

Reply:
thanks for making these more accurate/pulling them out 💯